junit-launcher/src/main/java/org/junit/gen5/launcher/main/Root.java

```diff
@@ -44,15 +50,13 @@ TestDescriptor getTestDescriptorFor(TestEngine testEngine) {
 	}
 
 	void applyPostDiscoveryFilters(TestDiscoveryRequest discoveryRequest) {
-		TestDescriptor.Visitor filteringVisitor = (descriptor, remove) -> {
-			if (!descriptor.isTest())
-				return;
-
-			if (isExcluded(discoveryRequest, descriptor)) {
+		Filter<TestDescriptor> postDiscoveryFilter = composeFilters(discoveryRequest.getPostDiscoveryFilters());
```

Comment:
Sorry @sbrannen , i think it was my monitor issue. The composeFilters has jumped down to next line. Its a browser formatting issue. (whats the max number of columns ? 100 ? 120?)